apps/src/templates/courseOverview/CourseOverview.js

```diff
@@ -38,7 +39,20 @@ const styles = {
   },
   versionDropdown: {
     marginBottom: 13,
-  }
+  },
+  selectSectionLabel: {
+    fontFamily: '"Gotham 5r", sans-serif',
+    fontSize: 14,
+    color: color.dark_charcoal,
+    marginTop: 15,
+    marginBottom: 4,
+  },
+  sectionSelector: {
+    width: 350,
+    height: 45,
+    fontSize: 18,
+    marginBottom: 15,
+  },
```

Comment:
probably fine, but would be good to share these styles with `ScriptOverview` since they're duplicate

Reply:
Good idea! I've extracted a shared component.